### PR TITLE
Fixes #4292 by removing Interface sections/blocks

### LINKED ISSUE #4292: Web IDL snippets on some pages
https://heycam.github.io/webidl/ is for browser implementers, but snippets of that syntax is included on some MDN pages.

Here are the ones I could find:
https://developer.mozilla.org/en-US/docs/Web/API/BluetoothAdvertisingData#interface
https://developer.mozilla.org/en-US/docs/Web/API/BluetoothDevice#interface
https://developer.mozilla.org/en-US/docs/Web/API/BluetoothRemoteGATTCharacteristic#interface
https://developer.mozilla.org/en-US/docs/Web/API/BluetoothRemoteGATTDescriptor#interface
https://developer.mozilla.org/en-US/docs/Web/API/BluetoothRemoteGATTServer#interface
https://developer.mozilla.org/en-US/docs/Web/API/BluetoothRemoteGATTService#interface
https://developer.mozilla.org/en-US/docs/Web/API/CSSGroupingRule#syntax
https://developer.mozilla.org/en-US/docs/Web/API/Microdata_DOM_API#htmlpropertiescollection

Searching for "webidl" and "web idl" case-insensitively might reveal more.

### PR DESCRIPTION
**Pull request by himanshugarg**
…us/web directory in archived-content repo

> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

Fixes #4292

> What was wrong/why is this fix needed? (quick summary only)

The interface blocks are apparently only for browser implementors and hence not required in mdn docs

> Anything else that could help us review it

1. I have not removed the following files as the directory en-us/web was not present in archived-content repo. 
2. Also I have removed the Interface section where nothing was left after removing the Interface code blocks.

### CODE REVIEW

File: files/en-us/web/api/mssitemodeevent/index.html [hunk 1]
```diff
@@ -116,10 +116,7 @@ <h3 id="Properties">Properties</h3>
 
 <h2 id="Example">Example</h2>
 
-<pre class="brush: js">interface MSSiteModeEvent extends Event {
-    buttonID: number;
-    actionURL: string;
-}
+<pre class="brush: js">
```
Comment (teoli2003) [inline files/en-us/web/api/mssitemodeevent/index.html:119]: This doesn't look like JavaScript…

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Yes, storing webidl inside pages was a very old attempt. Time to go.

Only, a place where the code doesn't look like JavaScript. If we don't know the language, I'm ok to just remove the brush.

File: files/en-us/web/api/mssitemodeevent/index.html (hunk 1 above)
Comment (himanshugarg) [reply files/en-us/web/api/mssitemodeevent/index.html:119]: Sure you mean `<pre class="js">`or just `<pre>`

Comment (teoli2003) [reply files/en-us/web/api/mssitemodeevent/index.html]: I mean: The content of the `<pre>` introduced by `<pre class="brush: js">` doesn't look like JavaScript.
So, either I'm mistaken (and it is JavaScript I don't know), or it is another language and we need to fix the brush we use.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**